package.json

```diff
@@ -24,29 +24,30 @@
   "main": "./lib/harvester.js",
   "dependencies": {
     "body-parser": "~1.4.3",
+    "debug": "^2.1.1",
     "express": "~4.6.1",
+    "i": "~0.3.2",
     "lodash": "~2.4.1",
+    "longjohn": "^0.2.4",
     "mkdirp": "~0.5.0",
     "mongodb": "^1.4.26",
     "mongojs": "^0.18.0",
     "mongoose": "^3.8.0",
     "rsvp": "~3.0.9",
-    "underscore.string": "^2.4.0",
-    "i": "~0.3.2",
-    "debug": "^2.1.1",
-    "longjohn": "^0.2.4"
+    "underscore.string": "^2.4.0"
   },
   "devDependencies": {
-    "mocha": "~1.20.1",
-    "should": "~4.0.4",
-    "supertest": "~0.13.0",
-    "http-as-promised": "^0.1.0",
-    "request-debug": "^0.1.1",
-    "nock": "^0.56.0",
     "chai": "^1.10.0",
     "chai-http": "^1.0.0",
+    "event-source-stream": "git+https://github.com/mavdi/event-source-stream.git",
+    "http-as-promised": "^0.1.0",
+    "mocha": "~1.20.1",
+    "nock": "^0.56.0",
+    "profanity-util": "0.0.2",
+    "request-debug": "^0.1.1",
+    "should": "~4.0.4",
     "sleep": "^1.2.0",
-    "profanity-util": "0.0.2"
+    "supertest": "~0.13.0"
```

Comment:
This is just the package.json being sorted alphabetically, not sure if supertest is used but it was in package.json before